code/array/array-new-vs-fixnum-times-map.rb

```diff
@@ -0,0 +1,17 @@
+require "benchmark/ips"
+
+ELEMENTS = 9
```

Comment:
I vote for link to this PR in `README.md`. 
This'll make info available for everyone, not only for people who reviewing PR's. Also, link will share that cool graph you have here:)